ui/app/styles/helper-classes/general.scss

```diff
@@ -77,6 +77,10 @@
   white-space: pre-wrap;
 }
 
+.overflow-wrap {
```

Comment:
did the `is-word-break` class not work for this situation? asking only because i was going to suggest renaming this class to `.break-word` to make it more obvious which CSS rule was being applied, but i see that basically already exists. 😛 

not a huge deal though since we could always add an `.overflow-wrap-anywhere` class helper down the line.

Reply:
Good question, and it didn't work. The white-space: pre-wrap on that helper borks things. Thanks for checking though!